target/linux/ath79/nand/base-files/etc/hotplug.d/firmware/10-ath9k-eeprom

```diff
@@ -19,6 +19,16 @@ case "$FIRMWARE" in
 		fi
 		ath9k_patch_mac $(macaddr_add $(mtd_get_mac_binary_ubi board-config 102) 1)
 		;;
+	meraki,z1)
+		. /lib/upgrade/nand.sh
+
+		if [ -n "$(nand_find_volume ubi0 caldata)" ]; then
+			caldata_extract_ubi "caldata" 0x1000 0x800
```

Comment:
I do wonder about this. nvmem can be used. I think when it fails it falls back to userspace.

Reply:
I tried on MR18 before, but it failed for me - so I'd rather get it submitted this way, then fix it.